src/bin/pg_autoctl/monitor.c

```diff
@@ -1448,59 +1448,62 @@ printCurrentState(void *ctx, PGresult *result)
 
 
 /*
- * monitor_print_state_as_json prints to given stream a single string that
- * contains the JSON representation of the current state on the monitor.
+ * monitor_print_last_events calls the function pgautofailover.last_events on
```

Comment:
is it possible to keep the function order. They are very similar looking functions with very small changes in the pr but since the ordering has changed, diff looks bigger than it is.